fedimint-bin-tests/src/main.rs

```diff
@@ -563,21 +565,368 @@ async fn cli_tests(dev_fed: DevFed) -> Result<()> {
         "{data_dir}/server-0/config-plaintext-2.json",
     );
 
-    fed.pegin(10_000).await?;
-    // fed.pegin_gateway(99_999).await?;
+    fed.pegin(10_000).await;
+    fed.pegin_gateway(99_999, &gw_cln).await;
 
-    let connect_string = fs::read_to_string(format!("{data_dir}/client-connect")).await?;
-    fs::remove_file(format!("{data_dir}/client.json"));
+    let connect_string = fs::read_to_string(format!("{}/client-connect", data_dir)).await?;
```

Comment:
I thought that `clippy` nags now if you don't use inline formatting form when poossible.